email/src/main/java/edu/jhuapl/dorset/demos/EmailManager.java

```diff
@@ -0,0 +1,363 @@
+package edu.jhuapl.dorset.demos;
+
+import java.io.IOException;
+import java.util.ArrayDeque;
+import java.util.Properties;
+
+import javax.mail.Flags;
+import javax.mail.Folder;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.Part;
+import javax.mail.Session;
+import javax.mail.Store;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class EmailManager {
+
+    private String user;
+    private String password;
+    private String mailStoreType = "imap";
+    private String host = "kossi-dev.jhuapl.edu";
```

Comment:
Remove this and then we'll squash the commits to remove evidence of it. We'll then add configuration to pass this in.